ui/client/cypress/integration/fragment.ts

```diff
@@ -44,7 +44,7 @@ describe("Fragment", () => {
     cy.wait(750)
     cy.get("[data-testid=window]").toMatchImageSnapshot()
 
-    cy.get("[data-testid=window]").contains("testOutput").parent().find("input").type("fragmentResult")
+    cy.get("[data-testid=window]").contains("testOutput").parent().find("input").clear().type("fragmentResult")
```

Comment:
shouldn't be `fragmentResult <dot>` ?

Reply:
Nope, here we put dedicated name for output. Below is `fragmentResult` with <dot>.